rero_ils/modules/items/serializers/response.py

```diff
@@ -52,7 +52,7 @@ def view(pid_fetcher, search_result, code=200, headers=None, links=None,
         tz = None
         if current_patron:
```

Comment:
`if current_patron and current_patron.get('libraries'):` (just to be sure)

Reply:
done